masterlist.yaml

```diff
@@ -28,10 +28,84 @@ globals:
         text: '[Postagem mais recente do LOOT](%1%).'
     subs: [ 'https://loot.github.io/latest-thread/' ]
 
-groups: []
+groups:
+- name: &dlcGroup DLC
+
+- name: &unofficialPatches Unofficial Patches
+  after:
+  - *dlcGroup
+
+- name: &default Default
```

Comment:
The name needs to be `default` for it to be recognised as the default group, it's case-sensitive.

Reply:
Fixed, thank you.